packages/glimmer-test-helpers/lib/environment.ts

```diff
@@ -507,6 +519,10 @@ class EmberishCurlyComponentManager implements ComponentManager<EmberishCurlyCom
     }
   }
 
+  didRenderLayout(component: EmberishCurlyComponent, bounds: Bounds) {
+    component.bounds = bounds;
```

Comment:
Yeah, this only need to _loosely_ emulate what Ember would do, and I think this is close enough. The ember version does use a symbol and pre-shape it in the constructor: https://github.com/emberjs/ember.js/commit/376a88df885fd4e88f39542988cdddd6021aded1